frontend/src/components/transactions/Transactions.tsx

```diff
@@ -25,7 +25,7 @@ class TransactionsWrapper extends Component<OuterProps> {
     count: number,
     paginationIndexer?: T.TxPagination
   ) => {
-    return await new TransactionsApi().getTransactions({
+    return await new TransactionsApi().getTransactionsFromIndexer({
```

Comment:
I suggest we rename `getTransactionsFromIndexer` to `getTransactions` to avoid confusion in the future (someone might read the name and wonder "where else can I get the list of transactions if not from the indexer?")